binderhub/static/vendor/xterm.js-2.6.0/xterm.css

```diff
@@ -1,2247 +0,0 @@
-    color: #080808;
-}
-
-.terminal .xterm-bg-color-232 {
-    background-color: #080808;
-}
-
-.terminal .xterm-color-233 {
-    color: #121212;
-}
-
-.terminal .xterm-bg-color-233 {
-    background-color: #121212;
-}
-
-.terminal .xterm-color-234 {
-    color: #1c1c1c;
-}
-
-.terminal .xterm-bg-color-234 {
-    background-color: #1c1c1c;
-}
-
-.terminal .xterm-color-235 {
-    color: #262626;
-}
-
-.terminal .xterm-bg-color-235 {
-    background-color: #262626;
-}
-
-.terminal .xterm-color-236 {
-    color: #303030;
-}
-
-.terminal .xterm-bg-color-236 {
-    background-color: #303030;
-}
-
-.terminal .xterm-color-237 {
-    color: #3a3a3a;
-}
-
-.terminal .xterm-bg-color-237 {
-    background-color: #3a3a3a;
-}
-
-.terminal .xterm-color-238 {
-    color: #444444;
-}
-
-.terminal .xterm-bg-color-238 {
-    background-color: #444444;
-}
-
-.terminal .xterm-color-239 {
-    color: #4e4e4e;
-}
-
-.terminal .xterm-bg-color-239 {
-    background-color: #4e4e4e;
-}
-
-.terminal .xterm-color-240 {
-    color: #585858;
-}
-
-.terminal .xterm-bg-color-240 {
-    background-color: #585858;
-}
-
-.terminal .xterm-color-241 {
-    color: #626262;
-}
-
-.terminal .xterm-bg-color-241 {
-    background-color: #626262;
-}
-
-.terminal .xterm-color-242 {
-    color: #6c6c6c;
-}
-
-.terminal .xterm-bg-color-242 {
-    background-color: #6c6c6c;
-}
-
-.terminal .xterm-color-243 {
-    color: #767676;
-}
-
-.terminal .xterm-bg-color-243 {
-    background-color: #767676;
-}
-
-.terminal .xterm-color-244 {
-    color: #808080;
-}
-
-.terminal .xterm-bg-color-244 {
-    background-color: #808080;
-}
-
-.terminal .xterm-color-245 {
-    color: #8a8a8a;
-}
-
-.terminal .xterm-bg-color-245 {
-    background-color: #8a8a8a;
-}
-
-.terminal .xterm-color-246 {
-    color: #949494;
-}
-
-.terminal .xterm-bg-color-246 {
-    background-color: #949494;
-}
-
-.terminal .xterm-color-247 {
-    color: #9e9e9e;
-}
-
-.terminal .xterm-bg-color-247 {
-    background-color: #9e9e9e;
-}
-
-.terminal .xterm-color-248 {
-    color: #a8a8a8;
-}
-
-.terminal .xterm-bg-color-248 {
-    background-color: #a8a8a8;
-}
-
-.terminal .xterm-color-249 {
-    color: #b2b2b2;
-}
-
-.terminal .xterm-bg-color-249 {
-    background-color: #b2b2b2;
-}
-
-.terminal .xterm-color-250 {
-    color: #bcbcbc;
-}
-
-.terminal .xterm-bg-color-250 {
-    background-color: #bcbcbc;
-}
-
-.terminal .xterm-color-251 {
-    color: #c6c6c6;
-}
-
-.terminal .xterm-bg-color-251 {
-    background-color: #c6c6c6;
-}
-
-.terminal .xterm-color-252 {
-    color: #d0d0d0;
-}
-
-.terminal .xterm-bg-color-252 {
-    background-color: #d0d0d0;
-}
-
-.terminal .xterm-color-253 {
-    color: #dadada;
-}
-
-.terminal .xterm-bg-color-253 {
-    background-color: #dadada;
-}
-
-.terminal .xterm-color-254 {
-    color: #e4e4e4;
-}
-
-.terminal .xterm-bg-color-254 {
-    background-color: #e4e4e4;
-}
-
-.terminal .xterm-color-255 {
-    color: #eeeeee;
-}
-
-.terminal .xterm-bg-color-255 {
-    background-color: #eeeeee;
-}
```

Comment:
So much was vendorized here, I’m glad this is switched to using packaging so it’ll be easier to review in the future.